### DISCUSSION TITLE
Do not write changes to the server and reboot on a fresh image

### DISCUSSION THREAD

**Discussion by at2f** [ltsp/ltsp#700] [Q&A]
Hello,

I am trying to create a setup in which multiple clients fetch the system's image, but no change is written back to the server (including in the home directory), and each time a machine is rebooted, it is rebooted with a fresh image.

This setup is meant public-facing machines in an internet café.

I tried adding `OMIT_IMAGE_EXCLUDES="home/*"` to `/etc/ltsp/ltsp.conf` under `[clients]` as mentioned [here](https://github.com/ltsp/ltsp/discussions/669#discussioncomment-2164777) but the changes are still written back to the server. I guess I misunderstood what was explained there.

Thanks in advance for your help.

**Comment by enaut** [comment]
The image is everything that is "part of the operating system". `/home/` is excluded per default. The home directories are instead mounted via sshfs or nfs depending on your settings. Everything in the image is readonly. So to get the "guest-account" you will need to replace the home mount.

**Reply by at2f** [reply]
Thank you for your answer.

I don't understand how I should do that, however. I've tried following [these explanations](https://ltsp.org/man/ltsp-nfs/) by adding `NFS_HOME=1` under `[server]`, and also `FSTAB_HOME="server:/home /home nfs defaults,nolock 0 0"` plus `OMIT_IMAGE_EXCLUDES="/home/*"` under `[clients]`, but it didn't change anything.

I also see that NFS is insecure and thus why it's better to be avoided for `/home`. Should I use sshfs instead?
